### PR TITLE
Add Clion folder to gitignore

### PR DESCRIPTION
**Pull request by heckad**

### CODE REVIEW

**Comment by an3l** [comment]
Hi @heckad ,

Thanks for your contribution.

Similar to other open source projects, the MariaDB Foundation needs to have shared ownership of all code that is included in the MariaDB distribution. The easiest way to achieve this is by submitting your code under the BSD-new license.

The other alternative is to sign the code contribution agreement which can be found here: https://mariadb.com/kb/en/mariadb/mca/

Please indicate in a comment below, or update your PR comment that you are contributing your new code of the whole pull request, including one or several files that are either new files or modified ones, under the BSD-new license or that you have filled out the contribution agreement and sent it.

Thanks,
Anel

File: .gitignore [hunk 1]
```diff
@@ -513,3 +513,6 @@ compile_commands.json
 .editorconfig
 .kateconfig
 *.kdev4
+
+# Clion && other JetBrains ides
+.idea
```
Comment (svoj) [inline .gitignore]: Could you please add newline at the end of the file?

**Comment by svoj** [comment]
@heckad could you also confirm that you're alright to contribute it either under terms of BSD-new or MCA?

**Comment by heckad** [comment]
Yes, I confirm that license is BSD-new.

**Comment by svoj** [comment]
Thanks!

**Pull request merged**